docs/CHANGELOG.asciidoc

```diff
@@ -46,6 +47,379 @@ Review important information about the {kib} 8.x releases.
+* AIOps: Adds dip support to log rate analysis in ML AIOps Labs ({kibana-pull}163100[#163100]).
+* AIOps: Improves table hovering for log rate analysis ({kibana-pull}162941[#162941]).
+* AIOps: Adds dip support for log rate analysis in observability alert details page ({kibana-pull}162476[#162476]).
+* Adds validation of field selected for log pattern analysis ({kibana-pull}162319[#162319]).
+* AIOps: Renames Explain Log Rate Spikes to Log Rate Analysis ({kibana-pull}161764[#161764]).
+* Adds new Data comparison view ({kibana-pull}161365[#161365]).
+* Adds test UI for text expansion models ({kibana-pull}159150[#159150]).
+* Adds update index mappings step to ML pipeline config workflow ({kibana-pull}163723[#163723]).
+Management::
+* Adds multiple formats for geo_point fields and make geo conversion tools part of field_format/common/utils ({kibana-pull}147272[#147272]).
+Maps::
+* Support time series split for top hits per entity source ({kibana-pull}161799[#161799]).
+Observability::
+* Adds support for group by to SLO burn rate rule ({kibana-pull}163434[#163434]).
+* Applying consistent design to Histogram and Custom Metric forms ({kibana-pull}162083[#162083]).
+* Adds preview chart to custom metric indicator ({kibana-pull}161597[#161597]).
+* Support filters for good/total custom metrics ({kibana-pull}161308[#161308]).
+Reporting::
+* Increase max size bytes default to 250mb ({kibana-pull}161318[#161318]).
+Security::
+* Change the default value of `session.idleTimeout` from 8 hours to 3 days ({kibana-pull}162313[#162313]).
+* User roles displayed on the user profile page ({kibana-pull}161375[#161375]).
+Uptime::
+* Implement private location run once ({kibana-pull}162582[#162582]).
+
+[float]
+[[fixes-v8.10.0]]
+=== Bug Fixes
+APM::
+* Fixes styling and port issue with new onboarding ({kibana-pull}163922[#163922]).
+* Fixes missing alert index issue ({kibana-pull}163600[#163600]).
+* Fixes trace waterfall loading logic to handle empty scenarios ({kibana-pull}163397[#163397]).
+* Fixes the action menu overlapping the 'Create custom link' flyout ({kibana-pull}162664[#162664]).
+* Improve service groups error messages and validations ({kibana-pull}162556[#162556]).
+* Fixes throwing appropriate error when user is missing necessary permission ({kibana-pull}162466[#162466]).
+* Hide components when there is no support from agents ({kibana-pull}161970[#161970]).
+* Fixes link to onboarding page in the Observability Onboarding plugin ({kibana-pull}161847[#161847]).
+Dashboard::
+* Disables top navigation actions when cloning or overlay is showing ({kibana-pull}162091[#162091]).
+Discover::
+* Fixes document failing to load when the ID contains a slash ({kibana-pull}160239[#160239]).
+* Fixes NoData screen in alignment with Dashboard and Lends behavior ({kibana-pull}160747[#160747]).
+* Fixes override parameter for create ({kibana-pull}160953[#160953]).
```

Comment:
I think it should be mentioned where the override parameter was fixed
```suggestion
* Fixes override parameter when creating data views ({kibana-pull}160953[#160953]).
```
What's more this is not a fix in Discover, it's part of the DataViews API

Reply:
Updated and moved to visualizations in commit fff14a4